packages/harmony/src/icons/UtilityIcons.stories.mdx

```diff
@@ -25,28 +25,32 @@ Utility icons are simple, single-color glyphs that identify labels and actions a
 <Heading>
 ## Usage
 
-Recommended sizes for utility icons: 14px | 16px | 20px | 24px | 30px | 32px
-
 </Heading>
 
 <Flex direction='column' gap='m'>
-  <Flex alignItems='center' gap='2xl'>
-    {Object.keys(iconSizes).map((size) => (
-      <Icons.IconNote key={size} size={size} />
-    ))}
-  </Flex>
+  Sizes
   <Flex alignItems='center' gap='2xl'>
     {Object.keys(iconSizes).map((size) => (
       <Flex
-        key={size}
+        direction='column'
         alignItems='center'
-        justifyContent='center'
-        h={iconSizes[size]}
-        w={iconSizes[size]}
-        css={{ background: 'rgba(255, 0, 0, 0.10)' }}
+        gap='s'
+        key={`container-${size}`}
```

Comment:
so i think we can just keep "size" as the key and remove the key for the flex item below